pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -56,37 +181,48 @@ class LightningTensor(Device):
 
     A device to perform tensor network operations on a quantum circuit.
 
+    This device is designed to simulate large-scale quantum circuits using tensor network methods. For
+    samll circuits, other devices like ``lightning.qubit``, ``lightning.gpu``or ``lightning.kokkos``  are
+    recommended.
+
+    Currently, only the Matrix Product State (MPS) method is supported, based on ``cutensornet`` backends.
+
     Args:
         wires (int): The number of wires to initialize the device with.
             Defaults to ``None`` if not specified.
+        max_bond_dim (int): The maximum bond dimension to be used in the MPS simulation.
         backend (str): Supported backend. Currently, only ``cutensornet`` is supported.
         method (str): Supported method. Currently, only ``mps`` is supported.
         shots (int): How many times the circuit should be evaluated (or sampled) to estimate
             the expectation values. Currently, it can only be ``None``, so that computation of
             statistics like expectation values and variances is performed analytically.
         c_dtype: Datatypes for the tensor representation. Must be one of
             ``np.complex64`` or ``np.complex128``.
-        **kwargs: keyword arguments. TODO add when cuTensorNet MPS backend is available as a prototype.
+        **kwargs: keyword arguments.
     """
 
     # pylint: disable=too-many-instance-attributes
 
     # So far we just consider the options for MPS simulator
     _device_options = ("backend", "c_dtype")
-
+    _CPP_BINARY_AVAILABLE = LT_CPP_BINARY_AVAILABLE
     _new_API = True
 
     # pylint: disable=too-many-arguments
     def __init__(
         self,
         *,
         wires=None,
+        max_bond_dim=128,
```

Comment:
What are the implications of setting the value to this?

Reply:
More explanation is added in line193-line 197